server/src/test/java/org/apache/druid/server/http/MetadataResourceTest.java

```diff
@@ -251,101 +260,124 @@ public void testGetAllSegmentsIncludingRealtime()
     Assert.assertEquals(new SegmentStatusInCluster(realTimeSegments[1], false, null, 40L, true), resultList.get(5));
   }
 
+
   @Test
-  public void testGetUnusedSegmentsInDataSource()
+  public void testGetUnusedSegmentsInDataSourceWithValidDataSource()
```

Comment:
this test could just be named `testGetUnusedSegmentsInDataSource` as it is a positive test anyway.

Reply:
```suggestion
  public void testGetUnusedSegmentsInDataSource()
```